lib/util/fs.js

```diff
@@ -46,23 +47,22 @@ function ensureDirectoryExists(dir) {
  * All found files, concatenated to the current dir
  */
 async function getFilepathsInFolder(dir, recursive = false) {
-  const results = [];
   const pathsList = await readdir(dir);
-  await Promise.all(
-    pathsList.sort().map(async (currentPath) => {
-      const currentFile = path.resolve(dir, currentPath);
-      const statFile = await stat(currentFile);
-      if (statFile && statFile.isDirectory()) {
-        if (recursive) {
-          const subFiles = await getFilepathsInFolder(currentFile, true);
-          results.push(...subFiles);
+  return flatten(
+    await Promise.all(
```

Comment:
I wonder if we even need a promise here in the first place. Since seeding is usually not done constantly on the runtime, but executed on-demand, it's not performance critical, so it might make sense to simplify implementation and make it sync.

Reply:
I agree your opinion!
But, this changing make `getFilepathsInFolder`, `Seeder._listAll` and `Seeder.run` to sync function from async function.

Can I change it?

Reply:
Great point. Double-checked how it is with Migrations, and they are async there, which is actually important, since there is also a MigrationSource abstraction there, which might be retrieving migrations e. g. from a database or other storage. Since we might eventually want to have same for Seeds, it's better to keep implementation consistent, so let's keep it the way it is.